P5/Source/Specs/persPronouns.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="persPronouns">
+  <gloss versionDate="2020-12-10" xml:lang="en">personal pronouns</gloss>
+  <desc versionDate="2020-12-10" xml:lang="en">lists personal pronouns.</desc>
```

Comment:
the verb "lists" might be misleading because in a TEI context I'd expect something like a list structure (of elements). And the desc is quite short, more or less repeating the gloss.
Maybe just borrow from the chapter prose, e.g. "indicates the personal pronouns used, or assumed to be used, by the individual being described."

Reply:
I agree with @peterstadler on expanding the desc

Reply:
Me too.